models/classes/class.ItemsService.php

```diff
@@ -68,6 +68,8 @@ class taoItems_models_classes_ItemsService extends OntologyClassService
 
     const INSTANCE_FORMAL_PARAM_ITEM_URI = 'http://www.tao.lu/Ontologies/TAODelivery.rdf#FormalParamItemUri';
 
+    const DIV_CLASS_EMPTY = '<div class="empty"';
```

Comment:
```suggestion
    private const DIV_CLASS_EMPTY = '<div class="empty"';
```

Reply:
because this is an old class i did not want to have visibility to this so either i will add visibility to all or i will leave this as is, please vote guys

Reply:
In our CR guideline we agreed to always add visibility for new constants and update the old ones if possible, so I would change, but https://github.com/oat-sa/extension-tao-item/pull/440/files#r526125972